src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -218,10 +223,11 @@ private void assertAllowedMimeType(final String mimeType) {
     }
   }
 
-  private void assertAllowedClassification(Attachment.Classification classification) {
-    if (DaoUtils.getEnumId(classification) != 0) {
-      // classification is not "undefined"
-      throw new WebApplicationException("Classification cannot be set", Status.FORBIDDEN);
+  private void assertAllowedClassification(final String classification) {
```

Comment:
The backend should work with the classification key, not the label. The label is only to present how that classification should be displayed to the user.
 
```suggestion
  private void assertAllowedClassification(final String classificationKey) {
```